torch/_inductor/overrides.py

```diff
@@ -1345,6 +1355,7 @@ def rand_like(x, **kwargs):
 computation_op_packed_map = {
     nn.Linear: packed_linear_eval,
     nn.Conv2d: packed_conv_eval,
+    nn.ConvTranspose2d: packed_conv_transpose_eval,
 }
```

Comment:
Not related to this PR but shall we introduce a configurable option (default to false) to disable weight prepacking? Just in case there is issue about weight prepacking that we want to triage with it.

Reply:
https://github.com/pytorch/pytorch/pull/93811 implements this feature.